test/audits/manifest/icons-192.js

```diff
@@ -40,7 +40,42 @@ describe('manifest: icons-192 audit', () => {
     return assert.equal(Audit.audit(inputs).value, false);
   });
 
+  it('fails when a manifest contains an icon with no size', () => {
+    const manifestSrc = JSON.stringify({
+      icons: [{
+        src: 'icon.png'
+      }]
+    });
+    const manifest = manifestParser(manifestSrc).value;
```

Comment:
that was intentional, because we have sane variable scope and to keep things parallel to other tests, but I can rename to something else